lib/trento_web/views/sap_system_view.ex

```diff
@@ -0,0 +1,157 @@
+defmodule TrentoWeb.SapSystemView do
+  use TrentoWeb, :view
+
+  def render("databases.json", %{databases: databases}) do
```

Comment:
I find the ordering hard to follow here. Maybe we could have `database` first and sap system after order. What do you think? Or does it break some other ordering rule?

Reply:
done!